x-pack/filebeat/input/netflow/input.go

```diff
@@ -89,10 +90,14 @@ func NewInput(
 	initLogger.Do(func() {
 		logger = logp.NewLogger(inputName)
 	})
-
+	conf := xpackInput.Config{}
+	if err := cfg.Unpack(&conf); err != nil {
+		return nil, err
+	}
 	out, err := connector.ConnectWith(cfg, beat.ClientConfig{
 		Processing: beat.ProcessingConfig{
 			DynamicFields: context.DynamicFields,
+			KeepNull:      conf.KeepNull,
```

Comment:
Looks like we have some copy and paste development here :)
Same as googlepubsub.